index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+  
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="portfolio.css">
+
+</head>
+<body>
+    <!-- TOOLBAR -->
+    <header class="toolbar">
+        <div class="logo"> Welcome </div>
+        <nav class="navbar"> <img src="images/icons/menu.png" alt="menu"> </nav>
+    </header>
+    <!-- END TOOLBAR -->
+
+    <!-- HOME -->
+    <section class="home">
+        <h2 class="home_title">
+            <span>Hey there. I’m Jihane</span>
+            <span class="home_title_green">I’m a software developer </span>
+        </h2>
```

Comment:
- [ ] I also suggest that the introduction content in the `<h2>` tag is put in an `<h1>`. This important because the `h1` is the star of our design - it captures the most attention so it should be emphasized. It can also be useful in case of search engine optimization.

Reply:
Done, thank you